pom.xml

```diff
@@ -55,8 +55,9 @@
         <dependency>
             <groupId>org.bukkit</groupId>
             <artifactId>bukkit</artifactId>
-            <version>1.14.4-R0.1-SNAPSHOT</version>
-            <scope>provided</scope>
+            <version>1.13.2</version>
+            <scope>system</scope>
+            <systemPath>${project.basedir}/deps/bukkit-1.13.2.jar</systemPath>
```

Comment:
Why the switch back to 1.13.2?

Reply:
- I did this because when I build build-tools,  it does not seem to provide me with the 1.14.4 bukkit jar
- Hmmm I may have misunderstood something with that process
- I will revert

Reply:
Reverted